ci-operator/step-registry/rosa-qe/sts/cluster/nightly/rosa-qe-sts-cluster-nightly-workflow.yaml

```diff
@@ -0,0 +1,24 @@
+workflow:
+  as: rosa-qe-sts-cluster-nightly
+  steps:
+    env:
+      ACCOUNT_ROLES_PREFIX: "rosa-sts-nightly"
+      CLUSTER_NAME: "sdq-nrosa-sts"
+      COMPUTE_MACHINE_TYPE: "m5.2xlarge"
+      MULTI_AZ: "true"
+      ENABLE_AUTOSCALING: "true"
+      MIN_REPLICAS: "3"
+      MAX_REPLICAS: "6"
+      CHANNEL_GROUP: "nightly"
+      ETCD_ENCRYPTION: "true"
```

Comment:
As we discussed offline, I leave an open question here, for rosa cluster, which one is better:
1) a flexible workflow, invoker can pass any supported parameter into it, e.g.  `ETCD_ENCRYPTION: true` or `ETCD_ENCRYPTION: false`
2) a fixed workflow, just like what we did in common OCP, the workflow reflects which features is enabled, e.g. `cucushift-installer-rehearse-aws-ipi-ovn`